modules/common/react/lib/utils/components.ts

```diff
@@ -166,7 +166,7 @@ export function useForkRef<T>(ref1: React.Ref<T>, ref2: React.Ref<T>): (instance
  *   return <Element ref={elementRef} {...elemProps} />
  * }
  */
-export function useLocalRef<T>(ref: React.Ref<T>) {
+export function useLocalRef<T>(ref?: React.Ref<T>) {
```

Comment:
I made these refs be optional in Typescript because they are technically optional. `strictFunctionTypes` surfaced this need.